src/github.com/getlantern/rotator/daily_rotator_test.go

```diff
@@ -1,10 +1,11 @@
 package rotator
 
 import (
-	"github.com/stretchr/testify/assert"
 	"os"
 	"testing"
 	"time"
+
+	"github.com/getlantern/testify/assert"
```

Comment:
It used to provide better printouts when Equal fails.  Not sure if that is
still true.

-- 
Cheers,
Ox

PGP Key - https://pgp.mit.edu/pks/lookup?op=get&search=ox@getlantern.org

-----------------------------------------------------------------------------------------

"I love people who harness themselves, an ox to a heavy cart,
who pull like water buffalo, with massive patience,
who strain in the mud and the muck to move things forward,
who do what has to be done, again and again."

- Marge Piercy